hcloud/instances.go

```diff
@@ -154,3 +221,80 @@ func nodeAddresses(addressFamily config.AddressFamily, networkID int64, server *
 	}
 	return addresses
 }
+
+func robotNodeAddresses(addressFamily config.AddressFamily, server *hrobotmodels.Server) []corev1.NodeAddress {
```

Comment:
Can both `hcloudNodeAddresses` and `robotNodeAddresses` be part of the genericServer Interface ? If so, let's postpone that for a future PR.

Reply:
For sure, Ill put it on my list.